cmd/ui/src/rendering/programs/edge.selfArrowHead.ts

```diff
@@ -0,0 +1,118 @@
+// Copyright 2024 Specter Ops, Inc.
+//
+// Licensed under the Apache License, Version 2.0
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * Sigma.js WebGL Renderer Arrow Program
+ * ======================================
+ *
+ * Program rendering direction arrows as a simple triangle.
+ * @module
+ */
+import CurvedEdgeArrowHeadProgram from './edge.curvedArrowHead';
+import { NodeDisplayData } from 'sigma/types';
+import { floatColor } from 'sigma/utils';
+import { Attributes } from 'graphology-types';
+import { bezier } from 'src/rendering/utils/bezier';
+import { getNodeRadius } from 'src/rendering/utils/utils';
+import { getControlPointsFromGroupSize } from './edge.self';
+
+const POINTS = 3,
+    ATTRIBUTES = 9,
+    STRIDE = POINTS * ATTRIBUTES;
+
+export default class SelfEdgeArrowHeadProgram extends CurvedEdgeArrowHeadProgram {
+    constructor(gl: WebGLRenderingContext) {
```

Comment:
same thing here

Reply:
Thanks for pointing this out! I removed these constructors.